src/cpu/aarch64/jit_uni_eltwise_int.cpp

```diff
@@ -0,0 +1,418 @@
+        const TReg &vr_to, const TReg &vr_from) {
+    assert(!"unsupported isa");
+}
+
+template <>
+void jit_uni_subkernel_int_t<sve_512>::process_relu(
+        const TReg &vr_to, const TReg &vr_from) {
+
+    scvtf(vr_from.s, p_all_one / T_m, vr_from.s);
+
+    fmul(vr_to.s, vr_from.s, ts_alpha);
+
+    fcmgt(p_mask.s, p_all_one / T_z, vr_from.s, t_zero.s);
+
+    sel(vr_to.s, p_mask / T_m, vr_from.s, vr_to.s);
+
+    frinti(vr_to.s, p_all_one / T_m, vr_to.s);
+    fcvtzs(vr_to.s, p_all_one / T_m, vr_to.s);
+}
+
+template <cpu_isa_t isa>
+void jit_uni_subkernel_int_t<isa>::store_8bit(const bool vectorize,
+        const XReg &mem_to, const TReg &vr_to, bool is_signed) {
+    assert(!"unsupported isa");
+}
+
+template <>
+void jit_uni_subkernel_int_t<sve_512>::store_8bit(const bool vectorize,
+        const XReg &mem_to, const TReg &vr_to, bool is_signed) {
+    if (vectorize) {
+        // store full TReg size
+        mov(t_tmp0.d, vr_to.d);
+        if (is_signed) {
+            smin(t_tmp0.s, 127);
+            smax(t_tmp0.s, -128);
+        } else {
+            umin(t_tmp0.s, 255);
+        }
+        st1b(t_tmp0.s, p_all_one, ptr(mem_to));
+    } else {
+        // store exactly one data item
+        // s32 save as s8/u8
+        mov(t_tmp0.d, vr_to.d);
+        if (is_signed) {
+            smin(t_tmp0.s, 127);
+            smax(t_tmp0.s, -128);
+        } else {
+            umin(t_tmp0.s, 255);
+        }
+        st1b(t_tmp0.s, p_mask_int8, ptr(mem_to));
+    }
+}
+
+} /* namespace */
+
+template <cpu_isa_t isa, data_type_t d_type>
+status_t jit_uni_eltwise_int_fwd_t<isa, d_type>::pd_t::init(engine_t *engine) {
+    bool ok = mayiuse(isa)
+            && desc()->data_desc.data_type == d_type
+            // only relu and linear so far
+            && utils::one_of(desc()->alg_kind, alg_kind::eltwise_relu,
+                    alg_kind::eltwise_linear)
+            && !has_zero_dim_memory()
+            && memory_desc_wrapper(src_md()).is_dense(true)
+            && attr()->has_default_values();
+
+    return ok ? status::success : status::unimplemented;
+}
+
+template <cpu_isa_t isa, data_type_t d_type>
+jit_uni_eltwise_int_fwd_t<isa, d_type>::jit_uni_eltwise_int_fwd_t(
+        const pd_t *apd)
+    : primitive_t(apd) {}
+
+template <cpu_isa_t isa, data_type_t d_type>
+status_t jit_uni_eltwise_int_fwd_t<isa, d_type>::init(engine_t *engine) {
+    const auto &desc = *pd()->desc();
+    CHECK(safe_ptr_assign(kernel_, new jit_uni_subkernel_int_t<isa>(desc)));
+    return kernel_->create_kernel();
+}
+
+template <cpu_isa_t isa, data_type_t d_type>
+jit_uni_eltwise_int_fwd_t<isa, d_type>::~jit_uni_eltwise_int_fwd_t() {
+    delete kernel_;
+}
+
+template <cpu_isa_t isa, impl::data_type_t d_type>
+void jit_uni_eltwise_int_fwd_t<isa, d_type>::execute_forward(
+        const exec_ctx_t &ctx) const {
+    auto src = CTX_IN_MEM(const data_t *, DNNL_ARG_SRC);
+    auto dst = CTX_OUT_MEM(data_t *, DNNL_ARG_DST);
```

Comment:
please use `CTX_OUT_CLEAN_MEM` to enforce proper zero padding.

Reply:
@mgouicem Thank you for the suggestion. We'll correct this PR next week, because the person in charge are taking vacation today.

Reply:
Thanks for the heads up!

Reply:
Resolving discussion because request for changes is addressed.